plugins/node/opentelemetry-instrumentation-mongodb/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Features
+
+* **mongodb:** support v6 ([#1666](https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1666))
+
```

Comment:
Changelog is automatically updated by the release automation. There is no need to do it here. It comes from the PR title.

Reply:
removed :)